scripts/shared/clusters.sh

```diff
@@ -304,13 +302,28 @@ function warn_inotify() {
     fi
 }
 
+# If any of the clusters use OVN-K as the CNI then clone the
+# ovn-kubernetes repo from master in order to access the required
+# kind scripts, and manifest generation templates.
+function download_ovnk() {
+    if [[ ${cluster_cni[*]} != *"ovn"* ]]
+    then
+        return
+    fi
+
+    echo "Cloning ovn-kubernetes from source"
+    rm -rf ovn-kubernetes/
+    git clone https://github.com/ovn-org/ovn-kubernetes.git
```

Comment:
```suggestion
    git clone https://github.com/ovn-org/ovn-kubernetes.git \
        || { git -C ovn-kubernetes fetch && git -C ovn-kubernetes reset --hard origin/master; }
```

On local envs this might be preferrable instead of cloning each time?

We could also use `git clone https://github.com/ovn-org/ovn-kubernetes.git || git -C ovn-kubernetes pull` for simplicity but if the remote branch diverges (e.g. force push) then it would fail, though on GH it's unlikely but is a **very remote** possibility.

Reply:
Yep, went with the first option, I doubt it will break anytime soon